core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/salt/update/SaltUpdateService.java

```diff
@@ -47,8 +50,12 @@ public void startingClusterServices(Long stackId) {
         flowMessageService.fireEventAndLog(stackId, UPDATE_IN_PROGRESS.name(), CLUSTER_RUN_SERVICES);
     }
 
-    public void rotateSaltPassword(Long stackId) {
-        flowMessageService.fireEventAndLog(stackId, UPDATE_IN_PROGRESS.name(), CLUSTER_SALT_PASSWORD_ROTATE_STARTED);
+    public void rotateSaltPassword(Long stackId, RotateSaltPasswordType type) {
+        stackUpdater.updateStackStatus(stackId, DetailedStackStatus.BOOTSTRAPPING_MACHINES);
```

Comment:
is it contains enough information for the user what the flow is doing? As I see, we use it more places, and it is the detailed information

Reply:
The event below contains additional info and it is written into the event history